examples/README.md

```diff
@@ -53,7 +53,8 @@ As an example, check out the TodoMVC example here: <https://examples.yew.rs/todo
 | [simple_ssr](simple_ssr)                           | F  | Demonstrates server-side rendering                                                                                                 |
 | [suspense](suspense)                               | F  | This is an example that demonstrates `<Suspense />` support                                                                        |
 | [function_memory_game](function_memory_game)       | F  | Implementation of [Memory Game](https://github.com/bradlygreen/Memory-Game)                                                        |
-| [timer](timer)                                     | S  | Demonstrates the use of the interval and timeout services                                                                          |
+| [timer](timer)                                     | S  | Demonstrates the use of the interval and timeout services using class components                                                                          |
+| [timer_functional](timer_functional)                                     | F  | Demonstrates the use of the interval and timeout services using function components                                                                          |
```

Comment:
Could you please make the functional timer example's spacing to align with other examples in the table?